src/styles/patientProfile.css

```diff
@@ -0,0 +1,52 @@
+.avatar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  border: ridge;
+  border-radius: 100%;
+  background-color: green;
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
lines 9, 10 and 11 in one line like that...
margin: 20px auto auto auto;

Reply:
yeah  you're right